### DISCUSSION TITLE
Unable to load MusicBox

### DISCUSSION THREAD

**Discussion by n8091977** [bareboat-necessities/lysmarine_gen#177] [Q&A] [answered]
Hello,

What a fantastic project!

I am after a little help as unable to gte MusicBox to load, the pertient errors I believe are listed below. Google has let me down, any help appreciated!

Thansk,
James

Jun  6 14:03:13 lysmarine budgie-panel.desktop[1988]: libGL error: failed to create dri screen
Jun  6 14:03:13 lysmarine budgie-panel.desktop[1988]: libGL error: failed to load driver: vc4
Jun  6 14:03:13 lysmarine budgie-panel.desktop[1988]: libGL error: failed to create dri screen
Jun  6 14:03:13 lysmarine budgie-panel.desktop[1988]: libGL error: failed to load driver: vc4

**Comment by mgrouch** [comment]
What image version are you using?
Can you show output of

cat /etc/bbn-build
?

**Reply by n8091977** [reply]
user@lysmarine:~ $ cat /etc/bbn-build
2022-05-09 21:47:22+01:00

**Comment by mgrouch** [comment]
Have tried 2023-05-27? I do not remember it having that issue. MusicBox and Iris are equivalent programs. You can use Iris instead of MusicBox. They are both just front-ends to mopidy.

**Reply by n8091977** [reply]
Thanks for the reply, what is the best way to upgrade without re-image?

**Reply by mgrouch** [reply]
> Thanks for the reply, what is the best way to upgrade without re-image?

No other supported way. You would need to load new image

**Comment by mgrouch** [comment, marked as answer]
Upgrade procedure is to copy off configuration of SD card somewhere else.
Load new image from new SD card. Apply stored configuration to a new loaded image.
